pkg/codegen/testing/test/testdata/configure-prop-names/python-extras/tests/test_codegen.py

```diff
@@ -38,16 +38,40 @@ def new_resource(self, args):
 
         :param MockResourceArgs args.
         """
-        assert args.inputs['bootDisk']['initializeParams']['image'] == "debian-cloud/debian-11"
+        assert args.inputs['bootDisk']['initializeParams']['imageName'] == "debian-cloud/debian-11"
         return 'foo', args.inputs
 
 @pulumi.runtime.test
 def test_func_with_default_value(my_mocks):
```

Comment:
Since we're adding code that now raises an error when a required arg is missing, can we have some runtime tests that test we get the expected error raised?

Reply:
Added